hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java

```diff
@@ -969,4 +972,8 @@ public static void validateBucket(OMMetadataManager metadataManager,
           BUCKET_NOT_FOUND);
     }
   }
+
```

Comment:
Please remove this method as no one is using it.

Reply:
Done